composer.json

```diff
@@ -47,7 +47,7 @@
         "drupal/rdf_entity": "dev-8.x-1.x",
         "drupal/redirect": "^1.0",
         "drupal/redis": "^1.0@RC",
-        "drupal/search_api": "~1.5",
+        "drupal/search_api": "1.6.0",
```

Comment:
This doesn't need to be locked any more.

Reply:
We're not in QA yes. Don't hurry ;)

Reply:
Woops sorry, I thought now that it is green it is ready for QA :) go ahead